R/escape.R

```diff
@@ -108,6 +108,17 @@ escape.integer64 <- function(x, parens = NA, collapse = ", ", con = NULL) {
   sql_vector(x, parens, collapse, con = con)
 }
 
+#' @export
+escape.raw <- function(x, parens = NA, collapse = ", ", con = NULL) {
+  sql_vector(sql_escape_raw(con, x), parens, collapse, con = con)
+}
+
+#' @export
+escape.blob <- function(x, parens = TRUE, collapse = ", ", con = NULL) {
+  pieces <- vapply(x, escape, character(1), con = con)
```

Comment:
Do we really need `escape.raw()`? Could we call into `sql_escape_raw()` directly instead?

```suggestion
  pieces <- vapply(x, sql_escape_raw, character(1), con = con)
```

Reply:
Thanks, removed escape.raw